.github/workflows/docker-bases.yml

```diff
@@ -101,6 +101,26 @@ jobs:
           build-args: 'arch=icx'
           tags: 'devitocodes/bases:cpu-icx'
 
+      - name: SYCL CPU image
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          file: './docker/Dockerfile.cpu'
+          push: true
+          target: 'cpu-icx-sycl'
```

Comment:
would drop the icx

Reply:
I put it there because SYCL is an open standard and there are multiple implementations... though yeah, the only that really matters at this point is Intel's, I think...